### ISSUE TITLE
Add content setting for adding teams to RPE

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#4311]
Add a content setting in the Events section for adding teams to an RPE. 

- ChAs should only be able to add teams to an event if the toggle is on
- If toggle is off, ChAs should be able to remove existing teams since teams drop up to the day of the event and those teams need to go back into the virtual judging pool

Related to #4140. Perhaps need to discuss if these two toggles interact at all.

**Comment by shaun-technovation** [comment]
Hope you don't mind, I created a new ticket for the admin controls (#4312), that work is standalone enough and a good chunk of work on its own I think.

**Comment by shaun-technovation** [comment]
I think this one and #4140 would work similarly to the registration flow - admins enable/disable registration via "Content & Settings" controls, there is an API to get these values that will control the registration profile types. Something similar would need to happen to get the admin settings for the new RPE controls (both registration and RPEs are using Vue.js).

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/4311?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by shaun-technovation** [comment]
Well, this is funny, we have a (poorly named) ENV setting, called `MANAGE_EVENTS` (which can be managed in Heroku to see how it works), that will show/hide the Manage Teams button based on this setting. One thing I don't particularly like about this functionality is that you can't see the current teams associated to the event (that I can find) when this setting is off; and I also don't like how it's a hidden, easy to forget about setting! Anywho, I vote we remove this functionality in another ticket. I'l proceed with the work here, I'll make the "Add Teams" button dynamically shown/hidden vs the entire section like the ENV does.

**Comment by shaun-technovation** [comment]
Also, regarding `MANAGE_EVENTS`, someone couldn't remove a team (part of the work in this issue) when this setting is off. Another reason why I think we should remove it.

**Comment by shaun-technovation** [comment]
I did the work for this one in the [PR](https://github.com/Iridescent-CM/technovation-app/pull/4322) for #4140, there was too much overlap for two separate PRs.

**Comment by shaun-technovation** [comment]
This one is on QA, if the corresponding admin control to prevent new teams from being added to an RPE (in #4312) is disabled, then the "ADD TEAMS" button should be hidden.

**Comment by dboyer** [comment]
Testing on QA:

- [ ] Turned off the toggle to allow adding teams and was not able to add teams to existing events. I was able to create a new event (the toggle to allow that was one) but could not add teams to the event

**Comment by dboyer** [comment]
@shaun-technovation Update! PT said they are okay with the adding team toggle also impacting whether or not judges can be added. No changes are needed.

**Comment by shaun-technovation** [comment]
> @shaun-technovation One issue is that it looks like turning off the "add teams" option also removes the ability to add judges. Chapters will likely need to continue adding judges during the RPE time period. Is there a way to adjust the allow adding teams toggle to not impact judges?

> @shaun-technovation Update! PT said they are okay with the adding team toggle also impacting whether or not judges can be added. No changes are needed.

Hmm, well, that's interesting. Glad everyone is okay w/ the functionality, it definitely wasn't intentional to make it this way, the toggle should only be for teams, not judges. We definitely can revisit this.

**Comment by dboyer** [comment]
Yeah I wondered if that was intentional. It's fine to leave it as is but if you'd prefer to have them as separate toggles since it wasn't intentional to have judges also impacted, I'm fine with taking the time to make that change.

**Comment by dboyer** [comment]
Tested by PT and no concerns found
